everestpy/src/everest/framework/__init__.py

```diff
@@ -1,3 +1,6 @@
 __version__ = '0.0.4'
 
-from .everestpy import *
+try:
+	from .everestpy import *
+except:
```

Comment:
We need to figure out this. Maybe we can fix it, so that the sou would be imported from the same place.

In the worst case, here we should only capture the import error.